app/controllers/nominations/request_nomination_invite_controller.rb

```diff
@@ -30,9 +30,7 @@ def receive_school
 
     session[:nomination_request_form] = @nomination_request_form.serializable_hash
 
-    if @nomination_request_form.school.cip_only?
-      redirect_to cip_only_request_nomination_invite_path
-    elsif !@nomination_request_form.school.eligible?
+    if !@nomination_request_form.school.eligible? && !@nomination_request_form.school.cip_only?
```

Comment:
Sorry, what's the intention for `cip_only` schools here?

That it behaves as if the school is eligible?

Maybe we should add a method like `can_access_service` on a school, that combines `eligible` and `cip_only`?

Reply:
Yes that is the intention